osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -445,6 +453,32 @@ public virtual void PopulateScore(ScoreInfo score)
             score.TotalScore = (long)Math.Round(ComputeFinalScore(ScoringMode.Standardised, score));
         }
 
+        /// <summary>
+        /// Populates the given score with remaining statistics as "missed" and marks it with <see cref="ScoreRank.F"/> rank.
+        /// </summary>
+        public void FailScore(ScoreInfo score)
+        {
+            if (Rank.Value == ScoreRank.F)
+                return;
+
+            Rank.Value = ScoreRank.F;
+            Rank.Disabled = true;
+
+            Debug.Assert(maximumResultCounts != null);
+
+            if (maximumResultCounts.TryGetValue(HitResult.LargeTickHit, out int maximumLargeTick))
+                scoreResultCounts[HitResult.LargeTickMiss] = maximumLargeTick - scoreResultCounts.GetValueOrDefault(HitResult.LargeTickHit);
```

Comment:
I think this (and the line below) should also consider the number of missed ticks. `maximumLargeTick - (scoreResultCounts.GetValueOrDefault(HitResult.LargeTickHit) + scoreResultCounts.GetValueOrDefault(HitResult.LargeTickMiss))` or something like that. Probably separate it into a separate variable as well.

Reply:
It overwrites the user's current misses, not add to it, so they shouldn't be accounted on filling the statistics (i.e. user's total miss = maximum hits - user's total hit). 

I was initially going to do it as you mentioned but realised I'm only making it complicated

Reply:
Right, fair enough.